### DISCUSSION TITLE
i really need help on something

### DISCUSSION THREAD

**Discussion by epicMulakk** [KadeArchive/Kade-Engine#2808] [Q&A]
so i have a liiittle problem when i build kade engine, i dont think anyone has posted this before
(its not my keyboard btw)
please help me

https://user-images.githubusercontent.com/97800428/149636420-492a2c58-4ea0-4c7b-b7c6-33195e316abd.mp4

**Comment by AnnoyingAlchemist** [comment]
Go into options and set a keybind for volume up/down

**Reply by epicMulakk** [reply]
alr ill try thanks

**Reply by epicMulakk** [reply]
it worked! thank you so much man
